crates/common/src/prov/model/to_json_ld.rs

```diff
@@ -1047,40 +1050,48 @@ trait Operate {
     fn derivation(&mut self, typ: &DerivationType);
 }
 
-impl Operate for JsonValue {
+impl Operate for Value {
```

Comment:
I'm surprised you kept this, not sure I would have implemented my own trait on a library type if I came at it now.

Reply:
Good point, I don't know what's idiomatic. I thought it an interesting approach at least, it took me a moment to realize what was happening.

I figured best to focus this PR on the changes that the library upgrade necessitated, we can refactor separately, this was already getting overly large.